common/xml/fpga_architecture.xsd

```diff
@@ -0,0 +1,667 @@
+<?xml version="1.0"?>

```

Comment:
Would you mind adding a comment which links to the VPR documentation for this file?